system-addon/lib/SearchFeed.jsm

```diff
@@ -0,0 +1,63 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+ /* globals ContentSearch, XPCOMUtils, windowMediator, Services */
+"use strict";
+
+const {utils: Cu} = Components;
+const {actionTypes: at, actionCreators: ac} = Cu.import("resource://activity-stream/common/Actions.jsm", {});
+const SEARCH_ENGINE_TOPIC = "browser-search-engine-modified";
+
+Cu.import("resource:///modules/ContentSearch.jsm");
```

Comment:
After looking through some other m-c files, looks like some aggressively lazily load everything that isn't used immediately for the first execution on loading the module. This would also include the `Actions.jsm` import. @k88hudson any suggestions?

E.g.,
https://dxr.mozilla.org/mozilla-central/source/toolkit/components/places/History.jsm
https://dxr.mozilla.org/mozilla-central/source/toolkit/components/places/PlacesUtils.jsm

But not always consistent:
https://dxr.mozilla.org/mozilla-central/source/toolkit/components/places/BookmarkHTMLUtils.jsm
https://dxr.mozilla.org/mozilla-central/source/toolkit/components/places/PlacesBackups.jsm